py/server/deephaven/table_listener.py

```diff
@@ -23,6 +23,7 @@
 _JListenerRecorder = jpy.get_type("io.deephaven.engine.table.impl.ListenerRecorder")
 _JPythonMergedListenerAdapter = jpy.get_type("io.deephaven.integrations.python.PythonMergedListenerAdapter")
 
+_DEFAULT_ON_ERROR_CALLBACK = lambda e : print(f"An error occurred during listener execution: {e}")
```

Comment:
This is not right. The error is more likely from an upstream table.

Reply:
In my experience, it is more likely from the listener code itself. But anyways, I actually dwelled on it a bit and eventually thought the message is general enough. But I am open to any suggestions.

Reply:
It's just wrong. Real data, not people twiddling with their listeners, has upstream failures. "My Kafka connection dropped." "I was wrong about the correctness of my join." "My formula had an NPE on some input."

"Update processing failure occurred" might be more accurate, if you're looking for a catch-all statement.

Reply:
I'm ok with @rcaudy suggestion